src/confluent_kafka/deserializing_consumer.py

```diff
@@ -26,34 +26,16 @@
 
 class DeserializingConsumer(_ConsumerImpl):
     """
-    A client that consumes records from a Kafka cluster. With deserialization
-    capabilities.
+    A high level Kafka consumer with deserialization capabilities.
 
-    Note:
+    Derived from the :py:class:`Consumer` class, overriding the :py:func:`Consumer.poll`
+    method to add deserialization capabilities.
 
-        The DeserializingConsumer is an experimental API and subject to change.
-
-    .. versionadded:: 1.4.0
-
-        The ``key.deserializer`` and ``value.deserializer`` classes instruct the
-        DeserializingConsumer on how to convert the message payload bytes to objects.
-
-    Note:
-
-        All configured callbacks are served from the application queue upon
-        calling :py:func:`DeserializingConsumer.poll`
-
-    Notable DeserializingConsumer configuration properties(* indicates required field)
+    Additional configuration properties:
 
     +-------------------------+---------------------+-----------------------------------------------------+
     | Property Name           | Type                | Description                                         |
     +=========================+=====================+=====================================================+
-    | ``bootstrap.servers`` * | str                 | Comma-separated list of brokers.                    |
```

Comment:
because there are a lot more of them as well, and the easiest consistent thing to do is to only specify the ones specific to this class here. this is not ideal though, as you point out.

important to note that i've also removed use of this class from the examples in this PR, and fully expect to deprecate it in the future - don't consider it that important.